### ISSUE TITLE
DB2 Porfiler errors out

### ISSUE THREAD

**Issue by TeddyCr** [open-metadata/OpenMetadata#6967]
```
[2022-08-25 23:35:49] WARNING  {Profiler:core:253} - Error trying to compute column profile for `colname` - (ibm_db_dbi.ProgrammingError) ibm_db_dbi::ProgrammingError: SQLNumResultCols failed: [IBM][CLI Driver][DB2/AIX64] SQL0418N  The statement was not processed because the statement contains an invalid use of one of the following: an untyped parameter marker, the DEFAULT keyword, or a null value.  SQLSTATE=42610 SQLCODE=-418
[SQL: /* {“app”: “OpenMetadata”, “version”: “0.11.5.0"} */
SELECT AVG(DOUBLE((col))) AS mean, count(col) AS “valuesCount”, count(DISTINCT col) AS “distinctCount”, min(col) AS “min”, NULL AS anon_1, max(col) AS “max”, NULL AS anon__1, sum(CASE WHEN (col IS NULL) THEN ? ELSE ? END) AS “nullCount”, STDDEV_POP(col) AS stddev, sum(col) AS “sum”
FROM “schema“.table FETCH FIRST 1 ROWS ONLY]
[parameters: (1, 0)]
(Background on this error at: https://sqlalche.me/e/14/f405)
```

**Comment by TeddyCr** [comment]
Currently working on a dev environment as M1 does not support DB2 drivers

**Comment by pmbrull** [comment]
@TeddyCr one possibility for us to try out is if we can install a later version of the package that supports M1 arch

**Comment by TeddyCr** [comment]
@pmbrull we are installing the latest releases of the packages. Unfortunately they do not support M1.

**Comment by TeddyCr** [comment]
Should be fixed at the same time than #6967
